pkg/asset/templates/kube-scheduler.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     k8s-app: kube-scheduler
     version: v1.3.6_coreos.0
 spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-scheduler
+  replicas: 2
```

Comment:
I believe we may achieve this by setting `stategy.rollingUpdate.maxUnavailable = 0`, right?